run.py

```diff
@@ -155,10 +160,10 @@ def evaluating(path, vocab_file, classifier_file, dictionary_file):
                             quad2 = image[:((rows)/2),(cols/2)+1:]
                             quad3 = image[(rows/2):,:((cols+1)/2)]
                             quad4 = image[(rows/2):,((cols+1)/2):]
-                        vector1 = __get_image_features_memory(quad1)
-                        vector2 = __get_image_features_memory(quad2)
-                        vector3 = __get_image_features_memory(quad3)
-                        vector4 = __get_image_features_memory(quad4)
+                        vector1 = __get_image_dense_features(quad1)
```

Comment:
@ZzimBoo91 Please use separate function to divide the image into quads. and use proper variable names (vector1, vector2 ...)